<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartMenueWidget</class>
 <widget class="QWidget" name="StartMenueWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Battle Isle Clone </string>
  </property>
  <widget class="QPushButton" name="qbutton_startGame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Spiel starten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton_loadGame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Spiel laden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton_options">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Optionen</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Eingestellte Optionen</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_opponent">
         <property name="text">
          <string>&lt;strong&gt;Gegner:&lt;/strong&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_opponent_indication">
         <property name="text">
          <string>Mensch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_limit">
         <property name="text">
          <string>&lt;strong&gt;Limit:&lt;/strong&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_limit_indication">
         <property name="text">
          <string>Keines</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_fogOfWar">
         <property name="text">
          <string>&lt;strong&gt;Fog of War:&lt;/strong&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_fogOfWar_indication">
         <property name="text">
          <string>Ein</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_map">
         <property name="text">
          <string>&lt;strong&gt;Karte:&lt;/strong&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_map_indication">
         <property name="text">
          <string>Erste</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;Battle Isle Clone&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
